### ISSUE TITLE
Instrument solo icon as (un)mute all button

### ISSUE THREAD

**Issue by BirchBirchBirch** [xrrhythmsuk/xruk-player#68]
A related thought was that it might be helpful if the headphone icon to the left of the solo buttons became a button to mute or unmute everyone (and if soloing is turned on, go back to everyone unmuted). Going off the original RoR player icons still used elsewhere, it would make sense to have a loudspeaker for everyone, a loudspeaker with an X for no-one (except whistle and shouting as above), and the headphones just for when soloing is on.

**Comment by timrog** [comment]
Maybe - I'm not sure what we gain from being able to mute everything though? You'd want to mute everything except one instrument, which it already does. Maybe there's some merit in having a reset button though?

**Comment by BirchBirchBirch** [comment]
True, a "mute all" would be more for symmetry than anything else. I agree a reset button would be helpful so users have another way to unmute a mixture of muted/unmuted instruments.
